### DISCUSSION TITLE
RSVP IP Address for geolocate

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#50] [General]
How can I record the ip address of a submitted rsvp? I'd like to get the geographic location of a submission for statistics.

**Comment by liedekef** [comment]
That's more your webserver's duty. EME doesn't store this (for privacy issues).

**Reply by roysinn** [reply]
Understandable. thanks for the quick answer.

**Reply by liedekef** [reply]
btw: there are hooks you can run after RSVP, so you can always create custom code to store the IP using $_SERVER['REMOTE_IP'] or so. See https://www.e-dynamics.be/wordpress/category/documentation/12-hooks-and-filters/ , the hook eme_insert_rsvp_action might be what you need then.

**Reply by roysinn** [reply]
Great! Many thanks Franky.
